src/components/job-row.tsx

```diff
@@ -175,13 +179,22 @@ export function buildJobRow(
   const cellContents: React.ReactNode[] = [
     <a onClick={() => showDetailView(job.job_id)}>{job.name}</a>,
     get_file_from_path(job.input_uri),
-    <OutputFiles
-      job={job}
-      openOnClick={(e: Event, output_uri: string) => {
-        e.preventDefault();
-        app.commands.execute('docmanager:open', { path: output_uri });
-      }}
-    />,
+    <>
+      {job.downloaded === false && job.status === "COMPLETED" && (
+        <IconButton aria-label="download" title="Download Outputs" onClick={async () => {
```

Comment:
"Outputs" doesn't look right here. Perhaps "Download Output" (collective noun) or "Download Output Files" (for consistency with the job list's column title) instead? This doesn't affect the function name.